lib/cisco_node_utils/cmd_ref/bgp.yaml

```diff
@@ -191,12 +191,13 @@ graceful_restart_timers_stalepath_time:
 
 log_neighbor_changes:
   kind: boolean
-  #TODO: Equivalent for bgp is 'bgp log'
-  # Currently supported but broken in IOS XR
+  default_value: false
+  cli_ios_xr:
+    config_get_token_append: '/^bgp log neighbor changes disable$/'
+    config_set_append: ' <state> bgp log neighbor changes disable'
```

Comment:
Seems like something isn't quite right here - so if I call `log_neighbor_changes = false`, then what gets configured on the box is `no bgp log neighbor changes disable` rather than `bgp log neighbor changes disable`?